azure-kusto-ingest/package.json

```diff
@@ -45,7 +45,6 @@
     "@types/uuid-validate": "0.0.1",
     "azure-kusto-data": "latest",
     "moment": "^2.22.2",
-    "request": "^2.88.0",
```

Comment:
How do you know it's safe to remove?

Reply:
Looks me like a left over and E2E passed